### PR TITLE
Scope project to SAUCE_USERNAME

### PR DESCRIPTION
**Pull request by dpgraham**
Adjustment to the visual examples to make it so that project is user specific:

Example: Change `project: Wdio examples` to `project: <username> Wdio examples` where username is SAUCE_USERNAME

Reason: For the Getting Started guide, we noticed that when a user runs a snapshot test after being invited to an org, it will use the snapshot test that was already there. We think it would be a better onboarding experience if the snapshot tests are specifically scoped to the user running the test so that they can go through the process of: creating a baseline, accepting a snapshot, updating the baseline, independently of the organization.

### CODE REVIEW

File: storybook/src/stories/Button.stories.ts [hunk 1]
```diff
@@ -38,13 +38,13 @@ export const Secondary: Story = {
 export const Large: Story = {
   args: {
     size: 'large',
-    label: 'Button',
+    label: 'Button!!!!!',
```
Comment (aluedeke) [inline storybook/src/stories/Button.stories.ts:41]: why do we need this?

**Review by aluedeke — APPROVED** [review]
LGTM, just curious what the !!!!! are about

File: nightwatch/cucumberjs/nightwatch.conf.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ module.exports = {
       },
       sauceVisualService: {
         buildName: 'Nightwatch Visual Demo Mocha Testrunner',
-        project: 'Nightwatch Mocha Testrunner Project',
+        project: `Nightwatch Mocha Testrunner Project for ${process.env.SAUCE_USERNAME}`,
```
Comment (wswebcreation) [inline nightwatch/cucumberjs/nightwatch.conf.js:61]:
```suggestion
        project: `Nightwatch CucumberJS Testrunner Project for ${process.env.SAUCE_USERNAME}`,
```

**Review by wswebcreation — APPROVED** [review]
LGTM, tnx

**Pull request merged**